operator/builtin/input/file/config.go

```diff
@@ -156,7 +157,7 @@ func (c InputConfig) Build(context operator.BuildContext) ([]operator.Operator,
 		InputOperator:         inputOperator,
 		Include:               c.Include,
 		Exclude:               c.Exclude,
-		SplitFunc:             splitFunc,
+		Multiline:             c.Multiline,
```

Comment:
Perhaps this is just a naming issue, but I'll have a difficult time accepting that the splitfunc is being delegated to a `Multiline` struct. Put another way, we need a split func regardless of whether or not we are using multiline splitting. 

I suspect that we can accomplish the same functionality as you are implementing, but by composing things in a better way.

Would it make sense to create a new struct which would pair the concept of a split func with the concept of a flush? Maybe something like:
```go
type Splitter struct {
    splitFunc // can be simple like '\n' or multiline config
    flusher     // forces a flush based on timing
}
```

Reply:
After looking at this further, I'm not convinced we even need to do what I've suggested here. Please see my related comment on `reader.go`. I'll leave this comment in place as a possible alternative, but I think we need to start by justifying the need to alter splitFunc behavior in any way.